### ISSUE TITLE
openstack_compute_instance_v2.VMNAME: unexpected EOF

### ISSUE THREAD

**Issue by ramonskie** [hashicorp/terraform#5244]
in terraform 0.6.11 we have the following error
see crash logs http://pastebin.com/0siPW5bU

problem does not exists in 0.6.10

**Comment by josdotso** [comment]
Having a similar issue on 0.6.11:

Full log is here:
https://gist.github.com/josdotso/2ace926c9405d1b19bf1

```
2016/02/23 19:35:14 [DEBUG] root: eval: *terraform.EvalIf
2016/02/23 19:35:14 [DEBUG] root: eval: *terraform.EvalSequence
2016/02/23 19:35:14 [DEBUG] root: eval: *terraform.EvalWriteStateTainted
2016/02/23 19:35:14 [DEBUG] root: eval: *terraform.EvalIf
2016/02/23 19:35:14 [DEBUG] root: eval: *terraform.EvalClearPrimaryState
2016/02/23 19:35:14 [DEBUG] root: eval: *terraform.EvalApplyPost
2016/02/23 19:35:14 [ERROR] root: eval: *terraform.EvalApplyPost, err: 1 error(s) occurred:

* openstack_compute_instance_v2.s1__instance_n1: Error waiting for instance (0afb79dd-5d95-409a-a402-cf932451e6f8) to become ready: Get https://openstack-cloud:8774/v2/393000acb7644ba78a15a4eadfbeabef/servers/0afb79dd-5d95-409a-a402-cf932451e6f8: EOF
2016/02/23 19:35:14 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error(s) occurred:

* openstack_compute_instance_v2.s1__instance_n1: Error waiting for instance (0afb79dd-5d95-409a-a402-cf932451e6f8) to become ready: Get https://openstack-cloud:8774/v2/393000acb7644ba78a15a4eadfbeabef/servers/0afb79dd-5d95-409a-a402-cf932451e6f8: EOF
2016/02/23 19:35:14 [ERROR] root: eval: *terraform.EvalOpFilter, err: 1 error(s) occurred:

* openstack_compute_instance_v2.s1__instance_n1: Error waiting for instance (0afb79dd-5d95-409a-a402-cf932451e6f8) to become ready: Get https://openstack-cloud:8774/v2/393000acb7644ba78a15a4eadfbeabef/servers/0afb79dd-5d95-409a-a402-cf932451e6f8: EOF
2016/02/23 19:35:14 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error(s) occurred:

* openstack_compute_instance_v2.s1__instance_n1: Error waiting for instance (0afb79dd-5d95-409a-a402-cf932451e6f8) to become ready: Get https://openstack-cloud:8774/v2/393000acb7644ba78a15a4eadfbeabef/servers/0afb79dd-5d95-409a-a402-cf932451e6f8: EOF
2016/02/23 19:35:14 [TRACE] Exiting eval tree: openstack_compute_instance_v2.s1__instance_n1
2016/02/23 19:35:14 [DEBUG] vertex provider.openstack (close), got dep: openstack_compute_instance_v2.s1__instance_n1
2016/02/23 19:35:14 [DEBUG] vertex provisioner.file (close), got dep: openstack_compute_instance_v2.s1__instance_n1
2016/02/23 19:35:14 [DEBUG] vertex root, got dep: provisioner.file (close)
2016/02/23 19:35:14 [DEBUG] vertex root, got dep: provider.openstack (close)
2016/02/23 19:35:14 [DEBUG] vertex root, got dep: provisioner.chef
2016/02/23 19:35:14 [DEBUG] waiting for all plugin processes to complete...
2016/02/23 19:35:14 [DEBUG] /home/vagrant/bin/terraform-provisioner-file: plugin process exited
2016/02/23 19:35:14 [DEBUG] /home/vagrant/bin/terraform-provisioner-local-exec: plugin process exited
2016/02/23 19:35:14 [DEBUG] /home/vagrant/bin/terraform-provisioner-remote-exec: plugin process exited
2016/02/23 19:35:14 [DEBUG] /home/vagrant/bin/terraform-provisioner-chef: plugin process exited
2016/02/23 19:35:14 [DEBUG] /home/vagrant/bin/terraform-provider-openstack: plugin process exited
```

**Comment by jtopjian** [comment]
Holy moly that's a long crash log :smile:  I'll see if I can figure out what the problem might be.

Would it be possible to share the terraform configuration you're using to trigger this error?

**Comment by jtopjian** [comment]
@ramonskie 0.6.12 was just released. Can you update and see if that fixes the crash you're seeing?

**Comment by jtopjian** [comment]
I just read through the crash log with glasses on and can confirm that this issue was resolved in 0.6.12. I'm going to close this issue, but please feel free to re-open it or open a new issue if I'm incorrect.
